snuba/writer.py

```diff
@@ -42,10 +43,11 @@ def write(self, rows):
 
 
 class HTTPBatchWriter(BatchWriter):
-    def __init__(self, schema, host, port, options=None):
+    def __init__(self, schema, host, port, options=None, table_name=None):
```

Comment:
It feels a little clunky that this overrides the schema attribute but I don't have any constructive ideas without giving it more thought.

Reply:
I think the only alternative way is to fit the temp_table into the dataset structure. Which defies the purpose of allowing the caller of the load script (ops generally) to handle the temporary table on their own.